JS/Lesson15/salih/script.js

```diff
@@ -105,6 +105,14 @@ checkAge(18);
 checkAge(65);
 
 // 3. Write a program that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+function checkIfTwoNumbers(num1, num2) {
+	return num1 > num2 ? " Number is Greater Then Second Number"
```

Comment:
This is called "ternary operator" ('condition ? result if true : result if false') https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator

It is often used to replace simple if-else statements however I'd say in this particular case it makes the code less readable. 
Still, I'm happy to see that you wanted to try it, keep it as is this time :)